tests/test_db_function.py

```diff
@@ -414,6 +415,26 @@ def test_retrieve_flag_for_application(db_session):
     assert result["user_id"] == second_flag.user_id
     assert result["flag_type"] == second_flag.flag_type.name
 
+def test_find_qa_complete_flag_for_applications(db_session):
+    """Put QA_COMPLETED flags in 1 out of 2 applications and
+    only retrieve the metadata for the 1 with QA_COMPLETED"""
+
+    first_flag = Flag(**flag_config[2])
```

Comment:
Can we use variable names for these flags with the flag type e.g. `qa_complete_flag` rather than `first_flag` etc. else it's a bit opaque what's going on?